postgres/14.dockerfile

```diff
@@ -5,4 +5,4 @@ ENV TZ="Europe/Berlin"
 
 COPY config /etc/postgresql
 
-HEALTHCHECK --interval=2s --timeout=5s --retries=10 CMD pg_isready
+HEALTHCHECK --interval=2s --timeout=5s --retries=10 CMD pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}
```

Comment:
```suggestion
HEALTHCHECK --interval=2s --timeout=5s --retries=10 CMD pg_isready -U $POSTGRES_USER -d $POSTGRES_DB
```

Reply:
In der docker-compose.yml musste ich `$$` verwenden. Ist es in der Dockerfile anders?

Reply:
Ahh du benutzt keine Klammern.